hooks/environment

```diff
@@ -36,7 +41,17 @@ if [[ "${BUILDKITE_AGENT_META_DATA_PLATFORM}" != "macos" ]]; then
   exit 4
 fi
 
-if [[ "${BUILDKITE_AGENT_META_DATA_HOSTNAME}" != "${permitted_signing_host}" ]]; then
-  echo "Host ${BUILDKITE_AGENT_META_DATA_HOSTNAME} is not permitted to do code-signing."
-  exit 5
+#if [[ "${BUILDKITE_AGENT_META_DATA_HOSTNAME}" != "${permitted_signing_host}" ]]; then
+#  echo "Host ${BUILDKITE_AGENT_META_DATA_HOSTNAME} is not permitted to do code-signing."
+#  exit 5
+#fi
```

Comment:
```suggestion
if [[ "${BUILDKITE_AGENT_META_DATA_HOSTNAME}" != "${permitted_signing_host}" ]]; then
  echo "Host ${BUILDKITE_AGENT_META_DATA_HOSTNAME} is not permitted to do code-signing."
  exit 5
fi
```

Reply:
Will need to set `permitted_signing_host="sean-mac-c02xv3a8jgh6"` to.. not Seans computer

Reply:
I've updated the permitted host to be `codesigning-agent`, since no-one (but sean?) gains anything having it be this